src/modules/detailsDVD.js

```diff
@@ -0,0 +1,105 @@
+      inputName.placeholder = 'Your name here';
+      const inputComment = document.createElement('input');
+      inputComment.placeholder = 'Your insights';
+      const commentBtn = document.createElement('button');
+      commentBtn.id = `commentBtn${i}`;
+      commentBtn.innerHTML = 'Comment';
+      const commentsWrapper = document.createElement('div');
+      commentsWrapper.classList.add('commentsWrapper');
+      const commentTitle = document.createElement('h2');
+      commentTitle.innerHTML = 'Comments';
+      const commentsData = async () => {
+        const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/zggEBXzpFcQqjDxvMhMz';
+        const connect = await fetch(
+          `${baseURL}/comments?item_id=commentBtn${i}`,
+          {
+            method: 'GET',
+          },
+        );
+        const response = await connect.json();
+        console.log(response.length);
+        if (connect.status === 200) {
+          commentTitle.innerHTML += `(${response.length})`;
+          for (let i = 0; i < response.length; i += 1) {
+            const entry = document.createElement('p');
+            entry.innerHTML = `${response[i].creation_date}---> ${response[i].username}:  ${response[i].comment}`;
+            commentsWrapper.appendChild(entry);
+          }
+        }
+      };
+      commentsData();
+      innerWrapper.appendChild(img);
+      innerWrapDetails.appendChild(summary);
+      innerWrapDetails.appendChild(duration);
+      innerWrapDetails.appendChild(rating);
+      innerWrapDetails.appendChild(website);
+      innerWrapper.appendChild(innerWrapDetails);
+      innerdetails.appendChild(innerWrapper);
+      innerWrapDetails.appendChild(commentsWrapper);
+      commentsWrapper.appendChild(commentSection);
+      commentSection.appendChild(commentsTitle);
+      commentsWrapper.appendChild(commentTitle);
+      commentSection.appendChild(inputName);
+      commentSection.appendChild(inputComment);
+      commentSection.appendChild(commentBtn);
+      closeBtn.addEventListener('click', () => {
+        details.classList.add('hidden');
+        innerWrapper.remove();
+        commentSection.remove();
+      });
+      commentBtn.addEventListener('click', async () => {
+        if (!inputName.value || !inputComment.value) {
+          alert('Not empty values allowed, please add your name and comment!');
+        } else {
+          const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/zggEBXzpFcQqjDxvMhMz';
+          const connect = await fetch(`${baseURL}/comments/`, {
+            method: 'POST',
+            body: JSON.stringify({
+              item_id: commentBtn.id,
+              username: inputName.value,
+              comment: inputComment.value,
+            }),
+            headers: { 'Content-type': 'application/JSON' },
+          });
+          await connect.text();
+          window.location.reload();
+        }
+      });
+    });
```

Comment:
- [ ] I see a repetition here. Following the DRY rule, Instead of writing the same code again, I recommend defining a method that is responsible for sending a comment to the API and calling that method in ` commentBtn.addEventListener`.